README.md

```diff
@@ -107,6 +107,38 @@ can be achieved using a compose file with the following entry:
 Where certificate and key paths need to be replaced with paths to your
 certificate and key files.
 
+### Token signing keys
+
+Mender backend uses JWT tokens for device and user authentication. The tokens
+are cryptographically signed and their authenticity is verified when making
+requests to the backend. Tokens are handed out
+by [Device Authentication](https://github.com/mendersoftware/deviceauth)
+and [User Administration](https://github.com/mendersoftware/useradm) services.
+
+Before starting mender in production, one needs to provide RSA private keys in
+PEM format for both services. This can be accomplished by generating key using
+openssl:
+
+```
+openssl genpkey -algorithm RSA -out key.pem -pkeyopt rsa_keygen_bits:2048
+```
```

Comment:
The original content in mender-docs lists this as the command to use:

```
openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256
```

What is the difference between the two? Are they interchangable? It look like yours is only generating the private key, and not a certificate.